.github/toolchain.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+set -e
+
+INSTALL_DIR=${1:-$HOME/toolchain}
+BASEURL=https://wk-contrib.igalia.com/yocto/meta-perf-browser/browsers/nightly/sdk
+FILE=wandboard-mesa/browsers-glibc-x86_64-core-image-weston-browsers-armv7at2hf-neon-wandboard-mesa-toolchain-1.0.sh
```

Comment:
Just a sidenote, the nightly backend contains versions from main/HEAD for the Webkit components (wpe, wpebackend-fdo and libwpe). There is also an toolchain with the stable versions: s/nightly/candidate/.

I'm sharing this info just in case you could be also interested in test the component against those stable versions as well.

Reply:
Oh, that's interesting, thanks for mentioning it! I think we should keep this PR small and only replace Travis. I have filed #272 to maybe work on improvements later.